client/src/Movies/MovieList.js

```diff
@@ -24,7 +25,9 @@ export default class MovieList extends Component {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <Link to={`/movies/${movie.id}`}>
+            <MovieDetails key={movie.id} movie={movie} />
+          </Link>
```

Comment:
This works fine, but React is throwing a warning here about your Links needing a unique key